apps/web/src/components/layout/components/OrganizationSelect.tsx

```diff
@@ -52,6 +53,21 @@ export default function OrganizationSelect() {
     setValue(currentOrganization?._id || '');
   }, [currentOrganization]);
 
+  useEffect(() => {
+    registerSpotlightActions(
+      (organizations || [])
+        .filter((item) => item._id !== value)
+        .map((item) => ({
+          id: 'change-org-' + item._id,
+          title: 'Change org to ' + capitalize(item.name),
+          onTrigger: () => {
+            switchOrg(item._id);
+          },
+        }))
+    );
+    removeSpotlightActions(['change-org-' + value]);
```

Comment:
Need to remove action so current org are not displayed, because it is shown before the org switch happens...